test/integration/lmtp/serverspec/lmtp_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe 'osl-imap::lmtp' do
+  it_behaves_like 'dovecot'
+end
+
+describe file '/etc/dovecot/dovecot.conf' do
+  its(:content) { should match(/^protocols = imap lmtp pop3$/) }
+end
+
+describe file '/etc/dovecot/conf.d/10-master.conf' do
+  its(:content) do
+    should match %r{service lmtp {
+  unix_listener /var/spool/postfix/private/dovecot-lmtp {
+    group = postfix
+    mode = 0600
+    user = postfix
+  }
+}}
+  end
+end
+
+describe file '/var/spool/postfix/private/dovecot-lmtp' do
+  it { should be_socket }
+  it { should be_mode 600 }
+  it { should be_owned_by     'postfix' }
+  it { should be_grouped_into 'postfix' }
+end
```

Comment:
Is there a way to test this service to ensure it's working properly?

Reply:
I've updated the `lmtp` suite to send an email to the `foo` user, and added serverspec tests that verify it was delivered using the LMTP transport.